src/lib/fetchChatter.ts

```diff
@@ -30,9 +30,9 @@ export class FetchChatter {
             'chatter');
     }
 
-    list(): Promise<any> {
```

Comment:
It's great that we're adding flexible REST based calls, I believe that we should keep the existing calls as they provide some easy to use methods for common chatter calls. Thinking of them like sidewalks we've built where we've seen well trodden grass.